com.civicrm.hrjobroles/js/test/services/filters.service.spec.js

```diff
@@ -0,0 +1,189 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/angular',
+  'mocks/data/job-role.data',
+  'common/angularMocks',
+  'job-roles/modules/job-roles.module'
+], function (angular, Mock) {
```

Comment:
`Mock` gives the impression that this is the Mock module, it's better to rename this variable to `mockJobRoles` or similar.